doc/man/8/ceph.rst

```diff
@@ -1194,11 +1194,11 @@ Usage::
 	{--no-increasing}
 
 Subcommand ``reweight-by-utilization`` reweight OSDs by utilization
-[overload-percentage-for-consideration, default 120].
+[overload-threshold, default 120 [max_weight_change, default 0.05 [max_osds_to_adjust, default 4]]] 
```

Comment:
the command description of this command reads

https://github.com/ceph/ceph/blob/240c46a75a44cb9363cf994cb264e9d7048c98a1/src/mgr/MgrCommands.h#L117-L123

i think the command definition for "oload"  is also correct. as it does specify the "overload-percentage".  but both "overload-percentage-for-consideration" and "overload-threshold" are confusing. as there is no reference of either of these words in the document or the output of the command.

how about rephrase it like

> Subcommand ``reweight-by-utilization`` reweighs OSDs by utilization. It only reweighs the OSDs whose utilization is above the specified percentage of the average. The percentage is 120 by default.

Reply:
My edit here was indeed trying to align with the wording in `MgrCommands.h`, but I get your point.  Since the idea is to improve the completeness entry, we should of course do so in the most clear manner we can (modulo the missing `t` in your proposal ;)  The entries in this file are by design concise, so we should resist the urge to explain in too much detail that is already found elsewhere.

Your wording seemed a little stilted to me too; I've pushed my own rewording, please review.